htdocs/commande/class/commande.class.php

```diff
@@ -3804,10 +3804,11 @@ function fetch($rowid)
 
     /**
      * 	Delete line in database
-     *
+     * 
+     *	@param   int  $notrigger	1 = disable triggers
      *	@return	 int  <0 si ko, >0 si ok
      */
-    function delete()
+    function delete($notrigger=0)
```

Comment:
Standard signature of delete function should be
* @param User $user User that delete
* @param int $notrigger 0=launch triggers after, 1=disable triggers
Can you introduce param $user before param $notrigger